packages/lab/stories/form-field-next/form-field-next.stories.tsx

```diff
@@ -18,60 +21,48 @@ export default {
 export const Skeleton: ComponentStory<typeof FormFieldNext> = (props) => {
```

Comment:
What's the idea behind the Skeleton? Do we need to keep it?

Reply:
it's to show the 'bare bones' of Form Field without any controls inside, and to show how the height is controlled by the controls so it 'collapses' - I think it may be a good idea to keep to understand that concept of having to provide controls